pkg/kf/commands/apps/push.go

```diff
@@ -225,5 +234,40 @@ func NewPushCommand(p *config.KfParams, pusher kf.Pusher, b SrcImageBuilder) *co
 		"Path to manifest",
 	)
 
+	pushCmd.Flags().IntVarP(
+		&instances,
+		"instances",
+		"i",
+		-1, // -1 represents non-user input
+		"the number of instances(default is 1)",
```

Comment:
nit: missing space

Reply:
Hi, I didn't get what do you mean. Our format is always this.

Reply:
```suggestion
		"the number of instances (default is 1)",
```